### DISCUSSION TITLE
PWS Display Issue - outage on Wunderground (Sept 20, 2022) - RESOLVED

### DISCUSSION THREAD

**Discussion by bdeprez** [cytech/Home-Assistant-wundergroundpws#105] [General]
Just in case others are going crazy to understand what's up with Wunderground not showing your PWS details, this is taken from their webpage:

![image](https://user-images.githubusercontent.com/22580735/191266945-65a6dcd3-3fc0-4a8d-9cdb-af768041b7ac.png)

Status above as of 9:12am EST.

**Comment by bdeprez** [comment]
Seems to be resolved now - 3:29pm EST

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
